src/SourceBuild/content/docs/package-table.md

```diff
@@ -0,0 +1,102 @@
+### Table
+
+--------------------------------------------------------------------------------------
+| Platform | main<br>(10.0.x&nbsp;Runtime) |
+| :--------- | :----------: |
+| **Windows x64** | [![][win-x64-badge-main]][win-x64-version-main]<br>[Installer][win-x64-installer-main] - [Checksum][win-x64-installer-checksum-main]<br>[zip][win-x64-zip-main] - [Checksum][win-x64-zip-checksum-main] |

```

Comment:
Version badges appear truncated - 
![image](https://github.com/user-attachments/assets/26a889e6-99ab-4e85-8cf9-fc7374c991a8)

Reply:
This gets fixed in this PR but requires a new official build. See the svg change in this PR.